crates/meilisearch/src/routes/indexes/settings.rs

```diff
@@ -25,6 +25,17 @@ use crate::Opt;
 /// It also generates a `configure` function that configures the routes for the settings.
 macro_rules! make_setting_routes {
     ($({route: $route:literal, update_verb: $update_verb:ident, value_type: $type:ty, err_type: $err_ty:ty, attr: $attr:ident, camelcase_attr: $camelcase_attr:literal, analytics: $analytics:ident},)*) => {
+        #[allow(dead_code)]
+        const _: () = {
+            #[allow(dead_code)]
+            const fn __verify_settings_exist() {
+                let _: fn() = || {
+                    let meilisearch_types::settings::Settings { $($attr: _,)* .. }:
```

Comment:
Hey @MichaScant, sorry for the delay, but, isn't it possible t remove the below `..`?
```suggestion
                    let meilisearch_types::settings::Settings { $($attr: _,)* }:
```

If we keep it, it will allow an addition in the settings struct that is not listed by this macro. 🤔

Reply:
I can however this would cause cargo test --locked --release --all to fail, as it is causing the following error: pattern requires `..` due to inaccessible fields.

I can get around this by using kind: _ , if that would work: 
let meilisearch_types::settings::Settings { $($attr: _,)* _kind: _ }:

Reply:
Yes you can try this, the goal is to ensure that a field added to the Settings is never forgotten in this macro, that's why we can't allow `..` 🤔